packages/runtime/test/consumption/attributes.test.ts

```diff
@@ -2665,6 +2671,90 @@ describe("DeleteAttributeUseCases", () => {
             expect(updatedPredecessor.deletionInfo?.deletionStatus).toStrictEqual(LocalAttributeDeletionStatus.DeletedByOwner);
             expect(CoreDate.from(updatedPredecessor.deletionInfo!.deletionDate).isBetween(timeBeforeUpdate, timeAfterUpdate.add(1))).toBe(true);
         });
+
+        test("should throw an error when deleting an OwnSharedAttribute when the Relationship is in status Pending", async () => {
```

Comment:
```suggestion
        test("should throw an error trying to delete an own shared Attribute when the Relationship is in status Pending", async () => {
```
Similar for the other tests.